code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -95,7 +95,7 @@
 
 /obj/item/weapon/gun/projectile/automatic/c20r
 	name = "\improper C-20r SMG"
-	desc = "A lightweight, fast firing gun, for when you REALLY need someone dead. Uses 12mm rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp"
+	desc = "A lightweight, fast firing gun. Uses 12mm rounds. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp."
```

Comment:
Also if you're gonna revert this, maybe consider making the `REALLY` into `<b>really</b>` for fancy styling.